src/Template/Bake/Element/Controller/add.ctp

```diff
@@ -18,7 +18,7 @@ $compact = ["'" . $singularName . "'"];
     /**
      * Add method
      *
-     * @return void Redirects on successful add, renders view otherwise.
+     * @return \Cake\Network\Response|void Redirects on successful add, renders view otherwise.
```

Comment:
Because void means that this method is not to be expected to return anything.
As soon as you sometimes return response that means you MUST check on the response.
Thus there is no logical point in adding |void. It must be |null now as the check reveals either null or not. never void. See also the other core functions doing it that way.